jupyter_book/commands/__init__.py

```diff
@@ -129,6 +158,21 @@ def build(path_book, path_output, config, toc, warningiserror, build):
                 {path_pdf_output_rel}\
             """
             )
+        if builder == "pdflatex":
+            print("Finished generating latex for book...")
+            print("Converting book latex into PDF...")
+            # Convert to PDF via tex and template built Makefile and make.bat
+            if sys.platform == "win32":
+                makecmd = os.environ.get("MAKE", "make.bat")
+            else:
+                makecmd = os.environ.get("MAKE", "make")
+            try:
+                with cd(OUTPUT_PATH):
+                    subprocess.call([makecmd, "all-pdf"])
+                print(f"A PDF of your book can be found at: {OUTPUT_PATH}")
```

Comment:
This should be a message box (see how success messages are generated elsewhere in jupyter book)

and the error below should be an `_error` box

Reply:
done